pkg/ebpf/ingress_node_firewall_events.go

```diff
@@ -57,12 +66,70 @@ func (infc *IngNodeFwController) ingressNodeFwEvents() error {
 				continue
 			}
 
+			// read the event header
+			if _, err := io.ReadFull(bytes.NewBuffer(record.RawSample), buf[:]); err != nil {
+				log.Printf("parsing perf event header err: %s", err)
```

Comment:
%s -> %v
Start print message with upper case

Reply:
... or %q if we want it quoted?